packages/grafana-ui/src/components/uPlot/config/UPlotSeriesBuilder.ts

```diff
@@ -1,9 +1,19 @@
 import tinycolor from 'tinycolor2';
 import uPlot, { Series } from 'uplot';
 import { DrawStyle, LineConfig, AreaConfig, PointsConfig, PointVisibility, LineInterpolation } from '../config';
-import { barsBuilder, smoothBuilder, stepBeforeBuilder, stepAfterBuilder } from '../paths';
 import { PlotConfigBuilder } from '../types';
 
+const pathBuilders = uPlot.paths;
```

Comment:
@leeoniya seems like the Plot tests are failing because of this. Are `paths` exposed from uPlot for sure?

Reply:
Is it possible this line is causing troubles https://github.com/leeoniya/uPlot/blob/a568fb12b6567567e9f5ce74a31d3628b56def5c/src/uPlot.js#L2346

Reply:
the only thing in the npm package is the `/dist` directory, and "module" in `package.json` points to the esm file in there, so:

https://github.com/leeoniya/uPlot/blob/530a46c53213bb63593451ae48d61e566185cb9e/dist/uPlot.esm.js#L3971-L3981